src/asmcnc/skavaUI/widget_xy_move.py

```diff
@@ -102,7 +106,7 @@
                         size: self.parent.size
                         pos: self.parent.pos  
                         Image:
-                            source: "./asmcnc/skavaUI/img/go_datum_y.png"
+                            source: pu.get_path('go_datum_y.png')
```

Comment:
Would be interested to see whether using get_path so often will cause a performance hit in the future (with existing searches, it's taking 4s on startup on a pi). Suggest we use the Kivy resources functions (see #code-focus) alongside some relative paths (yes, a pain to refactor but much quicker)

Reply:
Had this marked as "for later". Had a look at it now and it seems like a good idea to implement.